Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings

```diff
@@ -269,10 +269,10 @@
 "chat.drawing.settings.color.others" = "Escolha uma cor abaixo:";
 
 // Chat Typing
-"chat.users_are_typing" = "%@ are typing..."; // TODO
-"chat.user_is_typing" = "%@ is typing..."; // TODO
-"chat.typing" = "typing..."; // TODO
-"chat.several_typing" = "several users are typing..."; // TODO
+"chat.users_are_typing" = "%@ estão escrevendo...";
+"chat.user_is_typing" = "%@ está escrevendo...";
+"chat.typing" = "escrvendo...";
```

Comment:
Typo?

Reply:
those happen :stuck_out_tongue: 
will fix it now